python/ray/async_compat.py

```diff
@@ -11,3 +16,81 @@ def sync_to_async(func):
         return func(*args, **kwargs)
 
     return wrapper
+
+
+# Class encapsulate the get result from direct actor.
+# Case 1: plasma_fallback_id=None, result=<Object>
+# Case 2: plasma_fallback_id=ObjectID, result=None
+AsyncGetResponse = namedtuple("AsyncGetResponse",
+                              ["plasma_fallback_id", "result"])
+
+
+def get_async(object_id):
+    """Asyncio compatible version of ray.get"""
+    # Delayed import because raylet import this file and
+    # it creates circular imports.
+    from ray.experimental.async_api import init as async_api_init, as_future
```

Comment:
This codepath will be removed and shouldn't be used in this PR!

Reply:
Can you point out where will it be removed? Is it on-going PR?

Reply:
We will get rid of the pyarrow plasma client and serialization as soon as we can (i.e. once Python 2.7 is dropped). Ryans is working on backporting cloudpickle_fast to Python 3.5 and once that's done the `ray.experimental.async_api` is the only thing left using pyarrow.

Reply:
Can we get this PR in and modify the code path when we move async_api to raylet? I don't want this PR to be blocked by that.